commit-rights.md

```diff
@@ -0,0 +1,72 @@
+Committers Guidelines
+=====================
```

Comment:
```suggestion
Committers Guidelines for community.general
===========================================
```

Reply:
I assumed it is already specific enough since it is in the c.g repository - but you're right, it's better to be explicit :)